pkg/kube/kube_handler.go

```diff
@@ -229,7 +237,7 @@ type handler interface {
 	UpdateAppRepository(appRepoBody io.ReadCloser, requestNamespace string) (*v1alpha1.AppRepository, error)
 	RefreshAppRepository(repoName string, requestNamespace string) (*v1alpha1.AppRepository, error)
 	DeleteAppRepository(name, namespace string) error
-	GetNamespaces() ([]corev1.Namespace, error)
+	GetNamespaces(trustedNamespaces []corev1.Namespace) ([]corev1.Namespace, error)
```

Comment:
I like something like `presetNamespaces` or `whitelistedNamespaces`, I think that is the intention we want to convey.

Reply:
I like `whitelistedNamespaces`, this is exactly what we are doing.

Reply:
Thanks for all the changes! I personally do not think `whitelistedNamespaces` is a good name as it is against the [Inclusive Naming Initiative](https://inclusivenaming.org/language/word-list/) (the Linux Foundation, VMware and other organizations are supporting it). 
Therefore, I'd strongly suggest using an alternative name rather than this one: `allowListedNamespaced` or `allowedNamespaces` perhaps? See what you think @absoludity.

Reply:
Yep, +1 for inclusive naming. Naming is hard :) I think the same intention, in this specific case, is conveyed with `preapprovedNamespaces` or `precheckedNamespaces` ?